tests/Httpful/HttpfulTest.php

```diff
@@ -464,7 +464,7 @@ function testMultiHeaders()
         $req = Request::init();
         $response = new Response(self::SAMPLE_JSON_RESPONSE, self::SAMPLE_MULTI_HEADER, $req);
         $parse_headers = $response->_parseHeaders(self::SAMPLE_MULTI_HEADER);
-        $this->assertEquals('Value1,Value2', $parse_headers['X-My-Header']);
+        $this->assertEquals('Value1,Value2', $parse_headers['x-my-header']);
```

Comment:
Hmm. This means that this will be a breaking change. Code that relied on the capitalization will break. Can we keep this backwards compatible and keep the capitalization?